lib/static-cell/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "static-cell"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+armv6m-atomic-hack = {path = "../armv6m-atomic-hack"}
```

Comment:
Mentioned this in chat, but: for now, I'd be comfortable with you adding an armv6m-detector in `app/gimlet/src/tracing.rs` to prevent its copy-pasting into an armv6m application. Something like

```rust
if #[cfg(not(any(armv7m, armv8m)))] {
    compile_error!("ringbuf is unsound in the kernel on armv6m");
}
```

...phrased that way so that forgetting to include the `expose_m_profile` code in your build script won't cause false negatives.

Reply:
Fixed -- thank you!